src/routes/forumComments/editForumComment

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const mysql = require('mysql');
+const config = require('../../../config.js');
+
+//this edits an exising forum's information given the forum ID
+router.post('/api/editForumComment', (req, res) => {
```

Comment:
this method should be a `PUT` method instead of a `POST` method, you also need to change all calls to this API to use the `PUT` method instead, but everything else will remain the same

Reply:
I don't really understand the PUT method... can I stick with the post method, thanks!